packages/web/src/components/follow-artist-card/selectArtistsPreviewContext.tsx

```diff
@@ -120,12 +120,18 @@ export const SelectArtistsPreviewContextProvider = (props: {
       ? undefined
       : Math.min(30, Math.max(0, duration - 30))
 
-    dispatch(playerActions.play({ trackId: track_id, startTime, isPreview }))
     dispatch(
       playerActions.play({
         trackId: track_id,
         startTime,
-        isPreview,
+        playerBehavior: PlayerBehavior.PREVIEW_OR_FULL
```

Comment:
why is this not `FULL_OR_PREVIEW`? shouldn't this be the default?

Reply:
yeah this is a case where you're browsing artists from a grid to figure out who to follow, so I believe we do want previews here even if full tracks are available